src/tcp_mbedtls.c

```diff
@@ -231,7 +231,7 @@ int tcp_ssl_new_client(struct tcp_pcb *tcp, void *arg, const char* hostname, con
   mbedtls_ssl_config_init(&tcp_ssl->ssl_conf);
 
   mbedtls_ctr_drbg_seed(&tcp_ssl->drbg_ctx, mbedtls_entropy_func,
-                        &tcp_ssl->entropy_ctx, (const unsigned char*)pers, strlen(pers));
+                        &tcp_ssl->entropy_ctx, (const unsigned char*)pers, sizeof(pers));
```

Comment:
Huh? sizeof(pers) == sizeof(void *) == 4

Reply:
`void *` from where? `pers` is defined as `static const char pers[] = "esp32-tls";`.  sizeof(pers) will return the storage requirement of the entire array, which is 10. I believe this is different than say, `sizeof(const char *)`.

Reply:
Oops, you're right. However `sizeof(pers) == strlen(pers)+1` due to the null terminator. Not sure what is expected/needed here.

Reply:
right.. it is +1 due to '\0'.  Looks like there may not be a negative effect to this as the bytes are used for entropy seeding.